dev-tools/packaging/templates/windows/install-service.ps1.tmpl

```diff
@@ -12,3 +12,10 @@ $workdir = Split-Path $MyInvocation.MyCommand.Path
 New-Service -name {{.BeatName}} `
   -displayName {{.BeatName | title}} `
   -binaryPathName "`"$workdir\{{.BeatName}}.exe`" -c `"$workdir\{{.BeatName}}.yml`" -path.home `"$workdir`" -path.data `"C:\ProgramData\{{.BeatName}}`" -path.logs `"C:\ProgramData\{{.BeatName}}\logs`""
+
+# Attempt to set the service to delayed start using sc config.
+Try {
+  $svcDelayed='config "{0}" start=delayed-auto' -f '{{.BeatName}}'
```

Comment:
Would it work you directly put `{{.Beatname}}` instead of using `"{0}"`? Something like:

`Start-Process -FilePath sc.exe -ArgumentList "config","{{.BeatName}}","start=delayed-auto"`

Reply:
Yes, I'll update the PR with that